src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -372,18 +450,15 @@ public final class RegionName implements ResourceName {
 package com.google.cloud.simplecompute.v1;
 
 import com.google.api.core.BetaApi;
-import com.google.api.resourcenames.ResourceNameType;
+import com.google.api.resourcenames.ResourceNameFactory;
 import javax.annotation.Generated;
 
 @Generated("by GAPIC")
 @BetaApi
-public final class RegionNameType implements ResourceNameType {
-  private static RegionNameType instance = new RegionNameType();
-
-  private RegionNameType() {}
-
-  public static RegionNameType instance() {
-    return instance;
+public final class RegionNameFactory implements ResourceNameFactory {
```

Comment:
We don't need this given that a factory for `RegionName` can be acquired from `RegionName.newFactory()`, right?

Reply:
Right! Done.